admin/app.js

```diff
@@ -66,10 +66,9 @@ const index = require('./routes/index')
 const testDeveloper = require('./routes/test-developer')
 const serviceManager = require('./routes/service-manager')
 const school = require('./routes/school')
-const questions = require('./routes/questions')
-const pupilFeedback = require('./routes/pupil-feedback')
-const checkStarted = require('./routes/check-started')
-const completedCheck = require('./routes/completed-check')
+// const pupilFeedback = require('./routes/pupil-feedback')
```

Comment:
Can we remove the comments? Any particular reason for keeping them?

Reply:
I've commented them out as these `api` calls should no longer be used, therefore better security not to expose them.

We should raise a PR soon to delete this, an any other, dead code.